disvae/losses.py

```diff
@@ -205,16 +221,24 @@ def __call__(self, data, model, optimizer, storer):
         rec_loss = _reconstruction_loss(data1, recon_batch, storer=storer)
         kl_loss = _kl_normal_loss(*latent_dist, storer)
```

Comment:
this is where there's computation that is not used in case of `not is_mutual_information` just add a comment

Reply:
Added a TODO to remove once viz is fixed.